src/models/pie.js

```diff
@@ -41,11 +42,22 @@ nv.models.pie = function() {
         renderWatch.reset();
         selection.each(function(data) {
             var availableWidth = width - margin.left - margin.right
-                ,availableHeight = height - margin.top - margin.bottom
-                ,radius = Math.min(availableWidth, availableHeight) / 2
-                ,arcRadius = radius-(radius / 5)
-                ,container = d3.select(this)
+                , availableHeight = height - margin.top - margin.bottom
+                , radius = Math.min(availableWidth, availableHeight) / 2
+                , arcsRadiusOuter = []
+                , arcsRadiusInner = []
+                , container = d3.select(this)
                 ;
+            if (arcsRadius.length === 0) {
+                for (var i = 0; i < data[0].length; i++) {
+                    arcsRadiusOuter.push(radius - radius / 5);
```

Comment:
because you are pushing constant values in to the arrays, pre-calculate 'radius - radius/5', store it in a variable, and just push the variable